packages/store-indexer/src/sqlite/formatSqlQuery.ts

```diff
@@ -0,0 +1,50 @@
+import sqlParser from "node-sql-parser";
+
+type AstNode = {
+  type?: string;
+  [key: string]: unknown;
+};
+
+/**
+ * Transforms camelCase identifiers to snake_case in SQL queries
+ *
+ * @param sqlQuery The SQL query to transform
+ * @returns The transformed SQL query
+ */
+export function formatSqlQuery(sqlQuery: string): string {
```

Comment:
I feel like it might be clearer if we name this after what it does, like `normalizeColumnCase`